adserver/models.py

```diff
@@ -1567,7 +1602,8 @@ class PublisherPayout(TimeStampedModel):
     )
 
     class Meta:
-        ordering = ("-date",)
+        # This is 'date' instead of '-date' to make `first()` and `last()` work properly
+        ordering = ("date",)
```

Comment:
This requires a migration (why is a good question, but it does) and it will order the oldest payouts at the top of the page on the payouts screen. Do you think that's a little strange? I do realize that using `.last()` and it gets the first payout is a bit awkward too.

Reply:
I think we can just force the ordering on the page display if we want. The models should definitely default to time-based ordering I think.

Reply:
I pushed up this change.